packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2215,6 +2221,7 @@ export default class Meeting extends StatelessWebexPlugin {
         this.setLocalTracks(stream);
         this.mediaProperties.mediaDirection.sendVideo = sendVideo;
         this.mediaProperties.mediaDirection.receiveVideo = receiveVideo;
+        this.video = this.video || VideoStateMachine.create(this.mediaProperties.mediaDirection, this);
```

Comment:
Could you add the same comment from line 2174 about video being undefined?

`// video state could be undefined if you have not sent video before`